src/app/pages/user/user.component.ts

```diff
@@ -69,11 +69,11 @@ export class UserComponent implements OnInit, AfterViewInit {
       }
     });
   }
-  updateUser(_id: number, firstname: string, lastname: string, email: string, username: string) {
+  updateUser(_id: number, firstname: string, lastname: string, email: string, username: string, role: any, Class: any) {
```

Comment:
role and Class don't have type? Can you specify it?

Reply:
agree, at some point we will enforce `noExplicitAny` rule and we will need to fix it, thanks